.precious.toml

```diff
@@ -0,0 +1,53 @@
+[commands.prettier-scripts]
+type = "both"
+cmd = [
+  "npx",
+  "prettier",
+  "--parser",
+  "typescript"
+]
+lint-flags = ["--check"]
+tidy-flags = ["--write"]
+path-args = "absolute-file"
+include = ["src/**/*.ts"]
+ok-exit-codes = 0
+
+[commands.prettier-json]
+type = "both"
+cmd = [
+  "npx",
+  "prettier",
+  "--parser",
+  "json"
+]
+lint-flags = ["--check"]
+tidy-flags = ["--write"]
+path-args = "absolute-file"
+include = ["**/*.json"]
+ok-exit-codes = 0
+
+[commands.eslint-scripts]
+type = "both"
+cmd = [
+  "npx",
+  "eslint",
+  "--config",
+  "eslint.config.mjs",
```

Comment:
Do we need to specify the config file here? I think eslint automatically looks for the root eslint config file.